exec/java-exec/src/main/java/org/apache/drill/exec/store/dfs/easy/EasyFormatPlugin.java

```diff
@@ -115,6 +115,10 @@ public boolean isBlockSplittable() {
     return blockSplittable;
   }
 
+  /** Whether or not this format could also be in a compression container (for example: csv.gz versus csv).  
+   * This is an external compression container that would be handled outside of the format code. 
+   * This is an internal compression scheme, not supported by Parquet.
```

Comment:
A little unclear.  Maybe "If this format uses its own internal compression scheme, such as Parquet does, then this should return false."